generators/service-mongodb/templates/swift/instrumentation.swift

```diff
@@ -1,12 +1,13 @@
 import LoggerAPI
+import CloudEnvironment
 import MongoKitten
 
-var mongodb: Database!
-
-func initializeServiceMongodb() throws {
+func initializeServiceMongodb(cloudEnv: CloudEnv) throws -> Database {
+    let mongodb: Database
```

Comment:
If you move this declaration below the guard you won't need the type annotation since it would be inferred by the assignment.

Reply:
done